dist/projects.html

```diff
@@ -29,23 +29,23 @@
             <div class="row mb-5">
                 <div class="col-md-8 col-xl-6 text-center mx-auto">
                     <h2 class="fw-bold">Projects</h2>
-                    <p class="text-muted">Learn about the newest projects that Subsurface Studios has released</p>
+                    <p class="text-muted">Here are our latest and greatest projects! Check back later to see even more, or click on one of the projects to find out more!</p>
                 </div>
             </div>
             <div class="row row-cols-1 row-cols-md-2 mx-auto" style="max-width: 900px;">
                 <div class="col mb-4">
                     <div><a href="https://www.compensationvr.tk/download/"><img class="rounded img-fluid shadow w-100 fit-cover" src="assets/img/products/cvr.jpg" style="height: 250px;"></a>
                         <div class="py-4"><span class="badge bg-primary mb-2">Game</span>
                             <h4 class="fw-bold">Compensation Social</h4>
-                            <p class="text-muted">Compenstion Social is a virtual-reality social application centered around its fun games and strong creation tools</p>
+                            <p class="text-muted">Compenstion Social is a multiplayer social VR game centered around fun activities and letting your creativity run wild. From a variety of activities to a fully-featured creation suite, there's lots to see and even more to make.</p>
```

Comment:
Again, I felt it would be nice to make this a bit more personable. Once again, let me know if you want this change rolled back.